src/Microsoft.Azure.Relay/project.json

```diff
@@ -20,13 +20,16 @@
     "requireLicenseAcceptance": true
   },
   "frameworks": {
-    "net45": {
+    "net451": {
```

Comment:
@cesarme I had to update to at least 4.5.1 in order to use Microsoft.AspNetCore.WebUtilities package for parsing a query string.